### ISSUE TITLE
Add tool to query configured ports or ports in use for easy listing

### ISSUE THREAD

**Issue by HoorayJorge** [Qwinn1/forktools#4]
Often I just need a list of the ports currently being used, but its tedious to look them up.  This can be for Farmr, fd-cli or updating UFW for running the node publically.  Being able to query the list of a specific port type, like wallet_rpc or fullnode_port would be helpful.  Alternatively a CSV-style list or some other structure could be generated or all ports.

**Comment by Qwinn1** [comment]
Good idea.  Perhaps I'll implement this by allowing forkports to take a forkname as a parameter, in which case it'll display the ports for that single fork as you suggest before scanning for conflicts for just that fork.
